configure.ac

```diff
@@ -320,6 +319,46 @@ AS_CASE([$ENABLED_FIPS],
         AC_MSG_ERROR([Invalid value for --enable-fips "$ENABLED_FIPS" (main options: v1, v2, v5, ready, dev, rand, no, disabled)])
     ])
 
+# wolfEngine Options
+# This requires a FIPS source code bundle.
+AC_ARG_ENABLE([engine],
+    [AS_HELP_STRING([--enable-engine],[Enable wolfEngine options (default: disabled)])],
+    [ ENABLED_WOLFENGINE=$enableval ],
+    [ ENABLED_WOLFENGINE=no ]
+    )
+
+AS_CASE([$ENABLED_WOLFENGINE],
+    [no],[
+        ENABLED_WOLFENGINE="no"
+        ENABLED_WOLFENGINEWITHOUTFIPS="no"
```

Comment:
I would like to get rid of `ENABLED_WOLFENGINEWITHOUTFIPS`. I would just reduce `ENABLED_WOLFENGINE` to no, nofips, and fips. Then you can just do `test "$ENABLED_WOLFENGINE" != "no"` everywhere you've currently done `test "$ENABLED_WOLFENGINE" = "yes" || test "$ENABLED_WOLFENGINEWITHOUTFIPS" = "yes"`.